Editor/EditorCore/EditorHandleUtility.cs

```diff
@@ -519,14 +519,15 @@ static Vector3 GetBitangent(Vector3 planeNormal)
         /// Get a plane suitable for mouse input in a scene view.
         /// </summary>
         /// <param name="mousePosition"></param>
+        /// <param name="alignOnGrid"></param>
         /// <returns></returns>
-        internal static Plane FindBestPlane(Vector2 mousePosition)
+        internal static Plane FindBestPlane(Vector2 mousePosition, bool alignOnGrid = false)
         {
-            var res = FindBestPlaneAndBitangent(mousePosition);
+            var res = FindBestPlaneAndBitangent(mousePosition, alignOnGrid);
```

Comment:
@karl- 
I added a boolean to trace the Shape and Polyshape on the sceneview grid if visible rather than at a random height.
Is this boolean a necessity or should this be the default behavior ?
If I remember it was a remark from @gabrielw-us originally :)

Reply:
yes this should just be the default behaviour, no align on grid argument necessary